### ISSUE TITLE
Flake in FederatedDeployment e2e test

### ISSUE THREAD

**Issue by shashidharatd** [kubernetes-sigs/federation-v2#214]
`FederatedDeployment` e2e test seems to be flaking. here is one such failure. https://travis-ci.org/kubernetes-sigs/federation-v2/builds/417463481?utm_source=github_status&utm_medium=notification

```
Federated types "FederatedDeployment" resources 
  should be created, read, updated and deleted successfully
  /home/travis/gopath/src/github.com/kubernetes-sigs/federation-v2/test/e2e/crud.go:73
STEP: Obtaining a list of federated clusters
STEP: Checking readiness of cluster "cluster1"
Aug 17 22:07:51.416: INFO: Cluster cluster1 is Ready
Aug 17 22:07:51.423: INFO: Creating new FederatedDeployment in namespace "e2e-tests-federated-types-slg5f"
Aug 17 22:07:51.429: INFO: Created new FederatedDeployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn"
Aug 17 22:07:51.429: INFO: Creating new FederatedDeploymentPlacement in namespace "e2e-tests-federated-types-slg5f"
Aug 17 22:07:51.433: INFO: Created new FederatedDeploymentPlacement "e2e-tests-federated-types-slg5f/test-crud-6gxhn"
Aug 17 22:07:51.433: INFO: Creating new FederatedDeploymentOverride in namespace "e2e-tests-federated-types-slg5f"
Aug 17 22:07:51.438: INFO: Created new FederatedDeploymentOverride "e2e-tests-federated-types-slg5f/test-crud-6gxhn"
Aug 17 22:07:51.438: INFO: [Resource versions for e2e-tests-federated-types-slg5f/test-crud-6gxhn: template "713", placement "714", override "715"]
Aug 17 22:07:53.445: INFO: Waiting for Deployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn" in cluster "cluster1"
Aug 17 22:07:53.447: INFO: Updating FederatedDeployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn"
Aug 17 22:07:57.473: INFO: Waiting for Deployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn" in cluster "cluster1"
Aug 17 22:07:57.475: INFO: Updating FederatedDeploymentPlacement "e2e-tests-federated-types-slg5f/test-crud-6gxhn"
Aug 17 22:07:57.505: INFO: Waiting for Deployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn" in cluster "cluster1"
Aug 17 22:08:57.516: INFO: Timeout verifying Deployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn" in cluster "cluster1": timed out waiting for the condition
• Failure [66.108 seconds]
Federated types
/home/travis/gopath/src/github.com/kubernetes-sigs/federation-v2/test/e2e/crud.go:41
  "FederatedDeployment" resources
  /home/travis/gopath/src/github.com/kubernetes-sigs/federation-v2/test/e2e/crud.go:72
    should be created, read, updated and deleted successfully [It]
    /home/travis/gopath/src/github.com/kubernetes-sigs/federation-v2/test/e2e/crud.go:73
    Aug 17 22:08:57.516: Timeout verifying Deployment "e2e-tests-federated-types-slg5f/test-crud-6gxhn" in cluster "cluster1": timed out waiting for the condition
    /home/travis/gopath/src/github.com/kubernetes-sigs/federation-v2/test/e2e/framework/logger.go:39
```

**Comment by marun** [comment]
The only thing that has changed recently was the move to running e2e tests against a single node minikube cluster.  Depending on the environment the tests are run in, tests may take considerably longer due to the interaction between the federation and kubernetes controllers.  The fix is bumping the timeout further (see #205).

**Comment by marun** [comment]
Fixed by #253
